FeedReader/FeedReader.cs

```diff
@@ -74,27 +75,39 @@ public static HtmlFeedLink GetAbsoluteFeedUrl(string pageUrl, HtmlFeedLink feedL
         [Obsolete("Use GetFeedUrlsFromUrlAsync method")]
         public static IEnumerable<HtmlFeedLink> GetFeedUrlsFromUrl(string url)
         {
-            var task = GetFeedUrlsFromUrlAsync(url);
-            task.ConfigureAwait(false);
-            return task.Result;
```

Comment:
If the Task faults with an exception, Task.Result will throw an AggregateException. It is better to call Task.GetAwaiter().GetResult() since it will rethrow an "unwrapped" exception.